app/Http/Controllers/MoviesController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Controllers;
+
+//use App\Http\Requests\EditRequest;
+use App\Http\Requests\Movies\EditRequest;
+use App\Http\Requests\Movies\MoviesRequest;
+//use App\Http\Requests\MoviesRequest;
+use App\Models\Movies;
+use Illuminate\Http\Request;
+
+class MoviesController extends Controller
+{
+    public function createMovie()
+    {
+        return view('movies.create-form');
+    }
+
+    public function createCard(MoviesRequest $request)
+    {
+        $data = $request->validated();
+        $movie = new Movies($data);
+        $movie->save();
+
+        session()->flash('success', trans('messages.movie.success'));
+
+        return redirect()->route('movies.list');
+    }
+
+    public function list()
+    {
+        $movies = Movies::all();
+
+        return view('movies.list', ['movies' => $movies]);
```

Comment:
придерживаемся одного стиля
или через `compact(...)` или явно указываем массив
```suggestion
        return view('movies.list', compact('movies'));
```